### DISCUSSION TITLE
Screensaver

### DISCUSSION THREAD

**Discussion by Balter85** [moneytoo/Player#499] [Ideas]
After recently purchasing a Nvidia shield, and hating every single player I've tried... I finally found just player. It's probably the only reason I haven't returned my shield yet. Passes through all my audio, no sync issues, has hdr, DV.. LOVE IT.

My suggestion would be to please add a Screensaver feature. I have an OLED tv and worry about burn in. I pause my movies and walk away a lot. Would appreciate if the app took care of this. Thanks!

**Comment by moneytoo** [comment]
Isn't there a system screensaver already? https://www.nvidia.com/en-us/shield/support/shield-tv/sleep-timing-and-screen-saver/

**Reply by Balter85** [reply]
There is. The minimum kick on time is 5 min for the shield, which for an OLED is too long. 

The TV itself has a screensaver which kicks on in a minute or less but it doesn't apply to hdmi input sources. Only internal apps.
